httpstats/metrics.go

```diff
@@ -197,6 +202,15 @@ func (m *metrics) observeRequest(req *http.Request, op string, bodyLen int) {
 	m.http.method = req.Method
 	m.http.protocol = req.Proto
 	m.http.transferEncoding = transferEncoding
+
+	if m.options.pathObserved {
+		path := req.URL.Path
+
+		if strings.HasPrefix(path, "/") {
```

Comment:
How about doing it the other way around, and adding the leading `/` when it's missing? Seems like it's more natural to think of paths for HTTP requests as starting with a `/`.

Reply:
Seemed like it my be redundant when trying to filter by path in datadog and always adding the /. But cool by me.